mep/accounts/models.py

```diff
@@ -310,7 +310,7 @@ def __repr__(self):
     def __str__(self):
         details = self.account or self.person or ''
         if self.start_date or self.end_date:
-            details = '%s (%s)' % (details, '-'.join([
+            details = '%s (%s)' % (details, '–'.join([
```

Comment:
this case didn't include spaces previously, and if I understand it correctly, would need new logic to determine where to insert them depending on which parts of the range are missing. should I write this logic?

Reply:
I think just change the join string from `–` to ` – `

Reply:
i.e. add spaces around the en-dash (not obvious in my comment even though I used back ticks)